### PR TITLE
Support Decimal128 datatype

### PR DESCRIPTION
**Pull request by danslapman**

### CODE REVIEW

File: mongo_fdw.c [hunk 1]
```diff
@@ -1594,11 +1594,13 @@ ColumnValue(BSON_ITERATOR *bsonIterator, Oid columnTypeId, int32 columnTypeMod)
 		}
 		case NUMERICOID:
 		{
-			float8 value = BsonIterDouble(bsonIterator);
-			Datum valueDatum = Float8GetDatum(value);
+      bson_decimal128_t dec;
```
Comment (andreasscherbaum) [inline mongo_fdw.c]: Can you fix the spacing, please?

File: mongo_wrapper_meta.c [hunk 2]
```diff
@@ -281,6 +281,13 @@ BsonIterDouble(BSON_ITERATOR *it)
 }
 
 
+bool
+BsonIterDecimal128(BSON_ITERATOR *it, bson_decimal128_t *dec)
+{
+  return bson_iter_decimal128(it, dec);
```
Comment (andreasscherbaum) [inline mongo_wrapper_meta.c]: Can you please format this like the rest of the file?

Comment (ephemer) [reply mongo_wrapper_meta.c:287]: I think the issue is that the PR creator has used spaces (indentation level of 2) whereas the codebase uses tabs. Maybe the PR creator has their editor set up to display tabs at the width of two spaces

Comment (danslapman) [reply mongo_wrapper_meta.c:287]: sorry, i'll have to leave the pleasure to someone else

**Pull request closed without merging**